web/src/utils/ids.ts

```diff
@@ -0,0 +1,63 @@
+/* This file contains id generation reused by QE team
+ *  Please keep them updated of any change
+ */
+
+/**
+ * getTopologyGroupId gets a unique group id
+ * @param groupType string that represent group type.
+ * This could either be 'Node', 'Namespace' or any valid K8S Owner / Resource type
```

Comment:
FWI owner ids has also been simplified in https://github.com/netobserv/network-observability-console-plugin/pull/192/files#diff-ad9923146b0831f42e080b4b3e9edbdba8eae22114cea5a9b38cce6896f1e4d0L492 to match with this comment